src/components/map/StationPopup.js

```diff
@@ -54,7 +55,7 @@ export default function StationPopup({
 
       <div className="border-bottom py-1">
         <div className="fst-italic">
-          <VariableLabel variable={variable} />
+          <VariableLabel variable={variable} /> for {date.format("MMM YYYY")}
```

Comment:
Great addition! I noticed that when I switch to the Baseline display, the year that was set prior to switching is included in the popup.
Possibly from the [type](https://github.com/pacificclimate/weather-anomaly-tool/blob/2387a96aa96e30d805844eb2723dce00a78bdbca/src/components/map/DataMap.js#L127) part of the key?
That said, it's clear from the note: _"Note that baseline values are multi-year (1971-2000) averages for a given period (month)."_

Reply:
No, that's just an oversight. Well spotted. Fixed now.